test/integration/scheduler/predicates_test.go

```diff
@@ -30,450 +30,424 @@ import (
 
 // This file tests the scheduler predicates functionality.
 
-// TestInterPodAffinity verifies that scheduler's inter pod affinity and
-// anti-affinity predicate functions works correctly.
-func TestInterPodAffinity(t *testing.T) {
-	context := initTest(t, "inter-pod-affinity")
-	defer cleanupTest(t, context)
-	// Add a few nodes.
-	nodes, err := createNodes(context.clientSet, "testnode", nil, 2)
-	if err != nil {
-		t.Fatalf("Cannot create nodes: %v", err)
-	}
-	// Add labels to the nodes.
-	labels1 := map[string]string{
-		"region": "r1",
-		"zone":   "z11",
-	}
-	for _, node := range nodes {
-		if err = testutils.AddLabelsToNode(context.clientSet, node.Name, labels1); err != nil {
-			t.Fatalf("Cannot add labels to node: %v", err)
-		}
-		if err = waitForNodeLabels(context.clientSet, node.Name, labels1); err != nil {
-			t.Fatalf("Adding labels to node didn't succeed: %v", err)
-		}
-	}
+const pollInterval = 100 * time.Millisecond
 
-	cs := context.clientSet
-	podLabel := map[string]string{"service": "securityscan"}
-	podLabel2 := map[string]string{"security": "S1"}
+var podLabel = map[string]string{"service": "securityscan"}
```

Comment:
Please keep the labels and the test struct inside the test, as it was before. These are data for that particular test. We may add more tests to the file in the future. Besides, this change is unrelated to the title of this PR and makes the review hard.

Reply:
Understood. Let me cut this PR to the minimal to make this easy to review. Maybe a different PR to  reorg the code.